packages/postgraphile-core/__tests__/integration/schema/__snapshots__/defaultOptions.test.js.snap

```diff
@@ -176,7 +176,7 @@ type AnIntRangeBound {
   inclusive: Boolean!
 
   \\"\\"\\"The value at one end of our range.\\"\\"\\"
-  value: AnInt!
+  value: AnInt
```

Comment:
Changing an output type from non-null to nullable in GraphQL is a breaking change 😞